tests/test_models.py

```diff
@@ -0,0 +1,50 @@
+from pathlib import Path
+import pytest
+from nucleus import Dataset, DatasetItem, UploadResponse, Model, ModelRun
+from nucleus.constants import (
+    NEW_ITEMS,
+    UPDATED_ITEMS,
+    IGNORED_ITEMS,
+    ERROR_ITEMS,
+    ERROR_PAYLOAD,
+    DATASET_ID_KEY,
+)
+
+TEST_MODEL_NAME = '[PyTest] Test Model'
```

Comment:
Most of these should be imported from `https://github.com/scaleapi/nucleus-python-client/blob/master/tests/helpers.py`

Reply:
mo ved